room/common/src/main/java/androidx/room/ProvidedTypeConverter.java

```diff
@@ -16,21 +16,15 @@
 
 package androidx.room;
 
-import androidx.annotation.NonNull;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
 
 /**
- * Implementations of {@code TypeConverterFactory} interface are responsible to instantiate
- * TypeConverters.
+ * Marks a class as a type converter that will be provided to Room at runtime.
```

Comment:
```suggestion
 * Marks a class as a type converter that will be provided to Room at runtime.
 * If Room uses the annotated type converter class, it will verify that it is provided in the builder and if not, will throw
 * an exception.
```

Reply:
i think this got lost.